src/components/tasks/CreateTasks.jsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useRef, useState } from "react";
+import { useDispatch } from "react-redux";
+import { addTasks } from "src/store/actions/taskActions";
+import "src/components/tasks/CreateTasks.component.scss";
+import { createDate } from "src/utils/helpers/createDate";
+import PropTypes from "prop-types";
+
+const CreateTasks = ({ show, onShow }) => {
```

Comment:
You can name it as showTask or showButton for better understanding of what it represents.
```suggestion
const CreateTasks = ({ showTask, onShow }) => {
```